src/js/media-store/util.ts

```diff
@@ -85,6 +85,8 @@ export const toggleSubtitleTracks = (stateOwners, force: boolean): void => {
 export const areValuesEq = (x: any, y: any): boolean => {
   // If both are strictly equal, they're equal
   if (x === y) return true;
+  // If either is null, they're not equal
+  if (x == null || y == null) return false;
```

Comment:
added tests below for areValuesEq

Reply:
how so? we want check both on `null` and `undefined`. with the condition right above, if either is null or undefined we can safely say they are not equal...

Reply:
ah you're right since this is specifically for an early bail, we should be good. Resolving.